verticapy/tests/vDataFrame/test_vDF_plot.py

```diff
@@ -293,12 +295,10 @@ def test_vDF_bubble(self, iris_vd, titanic_vd,):
         assert max([elem[1] for elem in result3.get_offsets().data]) <= 7.9
         plt.close("all")
 
+    @pytest.mark.skipif(sys.version_info >= (3, 7), reason="this test is incompatible with newer versions of matplotlib")
     def test_vDF_density(self, iris_vd):
         # testing vDataFrame[].density
-        try:
-            create_verticapy_schema(iris_vd._VERTICAPY_VARIABLES_["cursor"])
-        except:
-            pass
```

Comment:
Passing an exception blinds us on any potential problem. We should try to avoid it.